README.md

```diff
@@ -1 +1,425 @@
+const feedFormatter = require('./formatter');
+
+class FeedKitties {
+
+	get mustHaveClient() {
+		return true;
+	}
+
+	get mustHavePayload() {
+		return true;
+    }
+
+    get struct() {
+        return struct.partial({
+            names: '[strings]',
+            food: 'string',
+            quantity: 'number'
+        });
+    }
+
+    async validate() {
+
+        if(this.data.quantity < 0)
+            throw new Error('Invalid Quantity');
+    }
+
+	async process() {
+
+        const kittyModel = this.session.getSessionInstance(KittyModel);
+
+        const kitties = await kittyModel.getBy('name', this.data.names);
+
+        await kittyModel.multiSave(feedFormatter(kitties, this.data));
+
+        return { message: 'Kitties Feeded' }
+	}
+}
+
+module.exports.handler = (...args) => Handler.handle(FeedKitties, ...args);
+```
+
+> :warning: For optimal use in local environments the FunctionName declare in the Hook should be the same in Lambda-Function Class
+
+#### Validation Error handling
+
+The `Handler` will not throw execptions if some validation stages failed, to avoid unnecesary retries. 
+
+In exchange it will log the error (you can watch it in *CloudWatch AWS service*), and the handler will response the `errorType` and `errorMessage`:
+
+```json
+{
+    "errorType": "LambdaError",
+    "errorMessage": "Invalid Client"
+}
+```
+
+But it can be changed if you need it overwriting `handleValidationError` method
+
+```js
+// in 'somewhere/Kitties/FeedKitties.js'
+'use strict';
+
+const { Handler } = require('@janiscommerce/lambda');
+const FeedKitties = require('./FeedKittiesFunction');
+
+class CustomHandler extends Handler {
+
+    // If you want to throw an Error
+    static handleValidationError(error) {
+        throw error;
+    }
+}
+
+module.exports.handler = (...args) => CustomHandler.handle(FeedKitties, ...args);
+```
+
+#### Process Errors Handling
+
+The `Handler` will throw exceptiosn during process stages (if erros occurs in `process` method), be carefull.
+
+#### Retries
+
+:warning:
+
+For `Async` executions the automatic retries in AWS services cannot be config with this package, you must do it manually or using other tools.
+
+#### Testing
+
+For testing you may do something like this:
+
+```js
+const sinon = require('sinon');
+const assert = require('assert');
+
+const FeedKitties = require('../somewhere/Kitties/FeedKitties');
+
+describe('Test', () => {
+
+    it('Should do something', async () => {
+
+        const event = { __clientCode: 'hiKitty', body: { names: ['Tom'], food: 'fish', quantity: 1 }}
+        assert.deepStrictEqual(await FeedKitties.handler(event), {
+            // something
+        });
+    });
+
+});
+
+```
+
+### :loudspeaker: Invoker
+
+The `Invoker` make **async*** invokes to a Lambda Function.
+
+#### CALL
+
+* `call(functionName, payload)` (*async*) : Invoke a function with a payload body. If multiple payloads are send make one invoke for payload.
```

Comment:
Yo lo refrasearía así: Invoke a function with a payload body. If payload is an array, it will make one invoke for each payload

Reply:
A lo que voy, siento que no está muy claro asi, es una opinión personal.